src/components/MeetupList.tsx

```diff
@@ -1,7 +1,7 @@
 // Libraries
 import React, { useState } from "react"
-import styled from "@emotion/styled"
 import { sortBy } from "lodash"
+import { Box, Flex, Input, Text } from "@chakra-ui/react"
 
 // Components
 import Emoji from "./OldEmoji"
```

Comment:
This one is deprecated. Lets use Emoji. https://github.com/ethereum/ethereum-org-website/blob/dev/docs/chakra-migration-guide.md#update-dependencies